addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2747,6 +2747,10 @@ const Many2oneUserValueWidget = SelectUserValueWidget.extend({
             return;
         }
         if (widget && widget === this.createButton) {
+            // When the create button is clicked, make sure the text
+            // value is restored from the actual input element because
+            // it might have been removed when hovering existing tags.
+            this.createInput._value = this.createInput.el.querySelector('input').value;
```

Comment:
All this createInput/createButton feel very hackish, we'll refactore that someday I hope :open_mouth: This fix is even more hacky although nice catch :+1: I am changing to not virtually emptying the create value on preview updates, I think it's less hackish and seems to work :+1:

Reply:
I tried to do that and did not manage to - I totally missed that `previewMode` was available in the event. :man_facepalming:

Reply:
@bso-odoo Nevermind, I made this: https://github.com/odoo/odoo/commit/edb3b1b6ad6d83263c2258cde74db57fbf327fc3, which works for the current blog usecase but I am not entirely sure it is ok for m2m widgets which could be used with previewMode enabled. I've restored what you did, with a TODO comment, we'll see later :)